### DISCUSSION TITLE
Can´t get Alpine 3 to work...

### DISCUSSION THREAD

**Discussion by zzseba78** [alpinejs/alpine#1482] [7. Bugs (archived)]
Can´t get Alpine 3 js to work.

I simplify it to a codepen where calls the cdn file ( https://unpkg.com/alpinejs@3.0.1/dist/cdn.min.js ), and a docs example, and thows errors.

https://codepen.io/zzseba78/pen/YzZdzRr

What i´m missing?

```
Uncaught TypeError: 'open' called on an object that does not implement interface Window.
Uncaught (in promise) ReferenceError: dropdown is not defined
Uncaught (in promise) ReferenceError: toggle is not defined...
```

**Comment by joshuahamilton** [comment]
I suspect in this case you'd need the `defer` attribute on the script tag, and I don't think CodePen will let you do that.
https://alpinejs.dev/essentials/installation#from-a-script-tag

If you try the simpler example here, it works: https://alpinejs.dev/directives/data
